modules/common/src/utils/utils.ts

```diff
@@ -161,20 +161,7 @@ export function batchArray<T> (items: T[], maxBatchSize: number): T[][] {
 }
 
 export function requiredEnvVariable (key: keyof typeof process.env): string {
-    if (key in process.env && process.env[key]) {
-        return <string> process.env[key];
-    }
-
-    if (Environments.isTest()) {
-        // enables running test directly in IDE in test mode
-        return '';
-    }
-
-    throw new Error(`Missing environment variable "${key}"!`);
-}
-
-export function requiredEnvProdVariable (key: keyof typeof process.env, envType: EnvironmentType): string {
-    if (envType !== EnvironmentType.PROD) {
+    if (key in NON_PRODUCTION_NULL_ENVIRONMENT_KEYS && process.env.NODE_ENV && !Environments.isProd()) {
```

Comment:
i would not do this. Then i would add a second paramater. This is much worse now than it was before.